tests/integration_tests/storage/test_field_parameter.py

```diff
@@ -178,7 +185,7 @@ def test_parameter_update_with_inactive_cells_xtgeo_grdecl(tmpdir):
 import os
 if __name__ == "__main__":
     if not os.path.exists("my_param.grdecl"):
-        values = np.random.standard_normal(4*4)
+        values = np.random.standard_normal(123*111*6)
```

Comment:
```suggestion
        values = np.random.standard_normal(NCOL*NROW*NLAY)
```

Reply:
Fixed this but needed a slightly different approach